### DISCUSSION TITLE
Addr-Seq +ASX : Trig/Gate Modded

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#264] [Ideas]
Hi,
I don't know if this allowed, difficult or not ;-) 
But this would be great:
ADDR-Seq trig/gate  (+ ASX extension). 
![ADDR-SEQ_TRIG_GATE_request](https://user-images.githubusercontent.com/105162700/174660083-07c1637b-77fc-4ceb-8272-4f224ea44d2f.jpg)
It's easy to modify a picture ! ...but a code...

**Comment by dromer** [comment]
This request should go to https://github.com/bogaudio/BogaudioModules/

We simply use their code and integrate it with Cardinal. So if you get them to create such new modules, then we can get them automatically as well!
